JunitHomework/src/main/java/string/utility/NumbersModifierNotOk.java

```diff
@@ -0,0 +1,34 @@
+package string.utility;
+
+public class NumbersModifierNotOk {
```

Comment:
this class naming is not ok. Class naming should be simple nouns. please read: http://www.oracle.com/technetwork/java/javase/documentation/codeconventions-135099.html#15411